### PR TITLE
feat(accordion): L3-3127 fixed accordions for Phillips Public

### PR DESCRIPTION
**Pull request by constantinehuzenko**
**Jira ticket**

[L3-3127](https://phillipsauctions.atlassian.net/browse/L3-3127)

**Screenshots**
| Before | After |
| ------ | ----- |
| NO VISUAL CHANGE | NO VISUAL CHANGE |

**Summary**

Changed format of accordion icons from image to svg

**Change List (describe the changes made to the files)**

- feat(src/components/Accordion/AccordionItem.tsx): changed format
- feat(src/components/Accordion/_accordion.scss): removed default svg behavior 

**Acceptance Test (how to verify the PR)**

- verify that open/close icons on accordion are the same as on https://phillips-seldon.netlify.app/?path=/docs/components-accordion--overview

**Regression Test**

- (Optional) Add verification steps to make sure this PR doesn't break old functionality

**Evidence of testing**

- Post logs, screenshots, etc

**Things to look for during review**

- [ ] PR title should correctly describe the most significant type of commit. I.e. `feat(scope): ...` if a `minor` release should be triggered.
- [ ] All commit messages follow convention and are appropriate for the changes
- [ ] All references to `phillips` class prefix are using the prefix variable
- [ ] All major areas have a `data-testid` attribute.
- [ ] Document all props with jsdoc comments
- [ ] All strings should be translatable.
- [ ] Unit tests should be written and should have a coverage of 90% or higher in all areas.

New Components

- [ ] Are there any [accessibility considerations](https://www.w3.org/WAI/ARIA/apg/patterns/) that need to be taken into account and tested?
- [ ] Default story called "Playground" should be created for all new components
- [ ] Create a jsdoc comment that has an Overview section and a link to the Figma design for the component
- [ ] Export the component and its typescript type from the `index.ts` file
- [ ] Import the component scss file into the `componentStyles.scss` file.

[L3-3127]: https://phillipsauctions.atlassian.net/browse/L3-3127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Accordion/AccordionItem.tsx [hunk 1]
```diff
@@ -52,30 +52,33 @@ const AccordionHeader = ({
...
   );
 
   const plusIconComponent = (
-    <img
-      className={getIconClasses(baseClassName, isLargeVariation, 'plus')}
-      src={plusIcon}
-      data-testid={`${id}-plusIcon`}
-      aria-hidden
-    />
+    <div>
```
Comment (adietrich3074) [inline src/components/Accordion/AccordionItem.tsx:65]: Is there a reason we need this wrapped in divs with no classes? The previous img tag didn't have a div tag so I would think the svg  component would work the same

**Review by adietrich3074 — APPROVED** [review]

**Review by scottdickerson — APPROVED** [review]
LGTM

File: src/components/Accordion/AccordionItem.tsx (hunk 1 above)
Comment (constantinehuzenko) [reply src/components/Accordion/AccordionItem.tsx:65]: yep, div is responsible for spacing

**Pull request merged**